avtozip/store/forms.py

```diff
@@ -12,19 +12,19 @@ class ProductForm(forms.ModelForm):
     fields = ('article', 'name', 'category', 'price', 'cost', 'count', 'store', 'is_active')
     widgets = {
         'article': forms.TextInput(
-            attrs={'placeholder': 'Article'},
+            attrs={'placeholder': _ul('Article')},
```

Comment:
Let us create generic approach for such cases.
Create the following base class with this implementation:
```python
class BaseModelForm(forms.ModelForm):
    """sss."""

    def __init__(self, *args, **kwargs):
        """sss."""
        for field_name, input in self.widgets.items():
            input.attrs['placeholder'] = self.model._meta.get_field(field_name).verbose_name
        super(BaseModelForm, self).__init__(*args, **kwargs)


class ProductForm(BaseModelForm):
    ...
```
Fill docstrings accordingly